docs/command_line_tutorial.md

```diff
@@ -0,0 +1,90 @@
+```
+
+The meaning of the additional arguments are as follows:
+
+- `--local` specifies not to use the Stanford CRFM API. You should always use `--local` unless you are a Stanford CRFM API user.
+- `--suite` specifies a subdirectory under the output directory in which all the output will be placed.
+- `--max-eval-instances` limits evaluation to only the first *N* inputs (i.e. instances) from the benchmark.
+
+`helm-run` creates an environment directory environment and an output directory by default.
+
+-  The environment directory is `prod_env/` by default and can be set using `--local-path`.
+-  The output directory is `benchmarking_output/` by default and can be set using `--output-path`.
+
+After running this command, navigate to the `benchmarking_output/runs/v1/` directory. This should contain a two sub-directories named `mmlu:subject=anatomy,model=huggingface_gpt-2` and `mmlu:subject=philosophy,model=huggingface_gpt-2`. Note that the names of these sub-directories is based on the run spec descriptions we used earlier, but with `/` replaced with `_`.
+
+Each output sub-directory will contain several JSON files that were generated during the corresponding run:
+
+- `per_instance_stats.json` contains a serialized list of `PerInstanceStats`, which contains the statistics produced for the metrics for each instance (i.e. input).
+- `run_spec.json` contains the `RunSpec`, which specifies the scenario, adapter and metrics for the run.
+- `scenario.json` contains a serialized `Scenario`, which contains the scenario for the run and specifies the instances (i.e. inputs) used.
+- `scenario_state.json` contains a serialized `ScenarioState`, which contains every request to and response from the model.
+- `stats.json` contains a serialized list of `PerInstanceStats`, which contains the statistics produced for the metrics, aggregated across all instances (i.e. inputs).
+
+The command also writes the file `benchmarking_output/runs/v1/run_specs.json`, which contains the run spec descriptions for all the runs. 
+
+`helm-run` provides additional arguments that can be used to filter out  `--models-to-run`, `--groups-to-run` and `--priority`. It can be convenient to create a large `run_specs.conf` file containing every run spec description of interest, and then use these flags to filter down the RunSpecs to actually run. As an example, the main `run_specs.conf` file used for the HELM benchmarking paper can be found [here](https://github.com/stanford-crfm/helm/blob/main/src/benchmark/presentation/run_specs.conf).
+
+## Using `helm-summarize`
+
+The `helm-summarize` reads the output files of `helm-run` and computes aggregate statistics. Run the following:
+
+```
+helm-summarize --suite v1
+```
+
+This reads the pre-existing files in `benchmark_output/runs/v1/` that were written by `helm-run` previously, and writes the following new files back to `benchmark_output/runs/v1/`:
+
+- `summary.json` contains a serialized `ExecutiveSummary` with a date and suite name.
```

Comment:
I think it'd be cool if every reference to a class (e.g., `ExecutiveSummary`, `Run`) has a GitHub hyperlink so people can click and see the schema.

Reply:
Good point, will add this.